com.dotsui.core/DotsUI.Core/LowLevelUtils.cs

```diff
@@ -13,65 +13,65 @@ namespace DotsUI.Core
 {
     public static unsafe class LowLevelUtils
```

Comment:
I'm not removing these functions completely. I'm thinking about backporting DotsUI to 2019.1/2019.2